js-control-structures/js-grades-excercise/js-grades-excercise.js

```diff
@@ -0,0 +1,32 @@
+const examPointsInput = prompt('Please add your points here');
+let grade;
+
+if (examPointsInput>=0 && examPointsInput<=100 && !examPointsInput.split('.')){
+    if(examPointsInput>=51 && examPointsInput<=60){
+        grade=6;
+        console.log(`Congratulations, you have passed the exam. Your grade is: ${grade}`);
+    }
+    else if(examPointsInput>=61 && examPointsInput<=70){
+        grade=7;
+        console.log(`Congratulations, you have passed the exam. Your grade is: ${grade}`);
+    }
+    else if(examPointsInput>=71 && examPointsInput<=80){
+        grade=8;
+        console.log(`Congratulations, you have passed the exam. Your grade is: ${grade}`);
+    }
+    else if(examPointsInput>=81 && examPointsInput<=90){
+        grade=9;
+        console.log(`Congratulations, you have passed the exam. Your grade is: ${grade}`);
+    }
+    else if(examPointsInput>=91 && examPointsInput<=100){
+        grade=10;
+        console.log(`Congratulations, you have passed the exam. Your grade is: ${grade}`);
+    }
+    else{
+        grade=5;
+        console.log('Sorry you have failed the exam');
+    }
```

Comment:
Here you just add one more IF condition to check the grade if it's > 5 then print the success message, else print the failed message